### DISCUSSION TITLE
Add a free 10 TB cloud | reff.cloud

### DISCUSSION THREAD

**Discussion by SingingDalong** [AlistGo/alist#3093] [Ideas]
reff.cloud is a free 10TB cloud. The server is unstable, but the API is open.
https://reff.cloud/api
I would love to be added to Alist.

**Comment by NovaShen555** [comment]
what about the speed?

**Reply by SingingDalong** [reply]
The upload speed is 5-8M, and the download speed is about 3-5M.

**Reply by NovaShen555** [reply]
too slow

**Reply by NovaShen555** [reply]
but i would like to try,after chinese new year
